packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -185,9 +198,25 @@ const DomainPicker: FunctionComponent< Props > = ( {
 					onChange={ handleInputChange }
 					onBlur={ onDomainSearchBlurValue }
 					value={ domainSearch }
+					disabled={ showErrorMessage }
 				/>
 			</div>
-			{ domainSearch.trim()?.length > 1 ? (
+			{ showErrorMessage && (
+				<div className="domain-picker__error">
+					<p className="domain-picker__error-message">
+						{ __( 'An error has occurred, please check your connection and retry.' ) }
+						{ domainSuggestionErrorMessage && ` ${ domainSuggestionErrorMessage }` }
```

Comment:
This won't be translated. You need to wrap the string with `__()`. But not here, where it's originally defined, the translation bot parses the code looking for phrases, but it doesn't evaluate it. So It has to be 

`__('the string itself')`

Reply:
Does the error message come from the API and is translated there already?

Reply:
For reference it does not look like the error messages are translated from the API already.